libcob/fileio.c

```diff
@@ -690,15 +690,17 @@ static unsigned int	bdb_lock_id = 0;
 	key.data = fld->data;			\
 	key.size = (cob_dbtsize_t) fld->size
 
-#if (DB_VERSION_MAJOR > 4) || ((DB_VERSION_MAJOR == 4) && (DB_VERSION_MINOR > 0))
-#define DBT_SET_APP_DATA(key,data)	((key)->app_data = (data))
+/* #if DB_VERSION_MAJOR >= 6 *//* while the fields are part of DBT since 4.1, it wasn't
+                             made part of the public API before 6.0 (DB_VERSION_FAMILY 12);
+                             older versions overwrite that, see bug NNN */
```

Comment:
please reference Bug 1032 here instead of NNN :-)

Reply:
Yes I didn't open it yet when I copied this and then forgot to edit it :sweat_smile: